templates/opp-single-form.php

```diff
@@ -5,40 +5,51 @@
  * This template is displayed immediately after the_content() is called within your theme file.
  * To adjust this template copy it into your current theme within a folder called "wivm".
  */
-$opp = new WI_Volunteer_Management_Opportunity( $post->ID ); //Get volunteer opportunity information
+$opp 			= new WI_Volunteer_Management_Opportunity( $post->ID ); //Get volunteer opportunity information
+$plugin_options = get_option( 'wivm-settings' );
 ?>
 
 <h3><?php ( $opp->opp_meta['one_time_opp'] == 1 ) ? _e( 'Sign Up to Volunteer', 'wired-impact-volunteer-management' ) : _e( 'Express Interest in Volunteering', 'wired-impact-volunteer-management' ) ; ?></h3>
 					
-<?php if( $opp->should_allow_rvsps() ): ?>
-<div class="loading volunteer-opp-message"><?php _e( 'Please wait...', 'wired-impact-volunteer-management' ); ?></div>
-<div class="success volunteer-opp-message"><?php _e( 'Thanks for signing up. You\'ll receive a confirmation email shortly.', 'wired-impact-volunteer-management' ); ?></div>
-<div class="already-rsvped volunteer-opp-message"><?php _e( 'It looks like you already signed up for this opportunity.', 'wired-impact-volunteer-management' ); ?></div>
-<div class="rsvp-closed volunteer-opp-message"><?php _e( 'We\'re sorry, but we weren\'t able to sign you up. We have no more open spots.', 'wired-impact-volunteer-management' ); ?></div>
-<div class="error volunteer-opp-message"><?php _e( 'Please fill in every field and make sure you entered a valid email address.', 'wired-impact-volunteer-management' ); ?></div>
+<?php if( $opp->should_allow_rvsps() ) { ?>
 
-<form id="wivm-sign-up-form" method="POST" url="<?php the_permalink(); ?>">
-	<?php wp_nonce_field( 'wivm_sign_up_form_nonce', 'wivm_sign_up_form_nonce_field' ); ?>
+	<div class="loading volunteer-opp-message"><?php _e( 'Please wait...', 'wired-impact-volunteer-management' ); ?></div>
+	<div class="success volunteer-opp-message"><?php _e( 'Thanks for signing up. You\'ll receive a confirmation email shortly.', 'wired-impact-volunteer-management' ); ?></div>
+	<div class="already-rsvped volunteer-opp-message"><?php _e( 'It looks like you already signed up for this opportunity.', 'wired-impact-volunteer-management' ); ?></div>
+	<div class="rsvp-closed volunteer-opp-message"><?php _e( 'We\'re sorry, but we weren\'t able to sign you up. We have no more open spots.', 'wired-impact-volunteer-management' ); ?></div>
+	<div class="error volunteer-opp-message"><?php _e( 'Please fill in every field and make sure you entered a valid email address.', 'wired-impact-volunteer-management' ); ?></div>
 
-	<?php do_action( 'wivm_start_sign_up_form_fields', $post ); ?>
+	<?php if( $plugin_options['use_gf'] != 1 ) { ?>
```

Comment:
Like we talked about when we first discussed this, I would much rather not adjust the core plugin specifically for this extension. Instead, let's figure out how to use hooks to allow us to adjust it with any extension.

I'm wondering if we can either add hooks in this file, or figure out a way to load up another template entirely.